arrayMethodsLab.js

```diff
@@ -0,0 +1,308 @@
+// # Exercises
```

Comment:
So your title of your project is correct ... Good job . You do have the logic, you have to work on the syntax of functions and how to implement the array method.